web/themes/custom/atrium/templates/node/node--srl-howto-instruction--full.html.twig

```diff
@@ -35,13 +35,80 @@
     </div>
   {% endif %}
 
-  {% set right_column = '' %}
-  {% if content.field_blocks|render or content.field_forms|render %}
-    {% set right_column %}
+  {% set right_column %}
+    <div class="mobile:display-none jcc-share--desktop">
+      {% include '@molecules/blocks/share/share.twig' with {
+        social_links: {
+          items: [
+            {
+              id: "print",
+              title: "Print",
+              src: "/libraries/courtyard/public/images/graphics/print.svg",
+              label: "Print current page",
+              renderer: 'patternlab',
+            },
+            {
+              id: "email",
+              title: "Email",
+              src: "/libraries/courtyard/public/images/graphics/email.svg",
+              label: "Email page to yourself",
+              uses_modal: true,
+              modal_content: drupal_entity('webform', 'mark_my_place'),
+              renderer: 'patternlab',
```

Comment:
What is `renderer` used for? I don't see that variable in the patternlab template.

Reply:
I thought this remedied an error that I encountered that occurs when displaying images within courtyard but maybe the error is not related. I will confirm and correct if needed.

Reply:
FYI... title does not seem to have been used at all in the template since first created:
https://github.com/JudicialCouncilOfCalifornia/courtyard/blob/a4530b059291609091854964171dd0b1b01daec9/source/_patterns/02-molecules/blocks/share/share.twig

Will remove from SRL-282 and file a to-do for courtyard to address.

Reply:
FYI... was not needed. Just something I did not circle back on when troubleshooting the twig error that was encountered. Most likely the error was probably a bad path or else incorrect string syntax.